REPOSITORY-GUIDELINES.md

```diff
@@ -0,0 +1,96 @@
+
+### ```Infra```
+
+Infrastructure repositories are those that are required to keep the Cilium project running smoothly, and are not designed to be used by adopters.
+
+Examples of repositories of this type include development and testing tools like [little-vm-helper](https://github.com/cilium/little-vm-helper), or those helping with the overall contribution process like [github-actions](https://github.com/cilium/github-actions).
+
+You can find the list of all ```Infra``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#infra-scope).
+
+### ```Special```
+
+Special repositories are non-archived repositories that do not fit neatly into ```Core```, ```Ecosystem```, or ```Infra```. They are generally not designed to be used by adoptors.
+
+They include the [Community](https://github.com/cilium/community) and [.github](https://github.com/cilium/.github) repository, template repositories like [design-cfps](https://github.com/cilium/design-cfps), as well as all forks.
+
+You can find the list of all ```Special``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#special-scope).
+
+### ```Historical```
+
+Historical repositories are considered deprecated, are no longer maintained, and are converted to an archived, read-only state. They are not recommended for use by adoptors.
+
+Repositories that have been inactive for quite some time may be good candidates for archival. They can be proposed through the same process as other [changes of scope](#change-of-scope).
+
+You can find the list of all ```Historical``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#historical-scope).
+
+## Lifecycle
+
+### Voting
+
+All changes that occur within the lifecycle of a repository have the same process:
+
+- An issue is opened via the appropriate issue template ([New Repository Template]() or [Change of Scope Template]())
+- Once the issue is opened, ample opportunity for discussion among both committers and the larger Cilium community is provided (for example, 1 week)
+- If there are no objections among committers during this time and any relevant criteria are met (for example, the criteria found in [Addition](#addition)), the change is approved
+- If there are any objections from committers, the committers decide if the change should be approved via a simple majority vote
+
+### Addition
+
+New repositories of any type can be proposed by using the [New Repository Template]() and can be approved via the process as described in [Voting](#voting).
+
+In order for a new repository to be approved, it must also meet the following criteria:
+
+- Approved by at least two committers, with the assumption that any committer who approves is taking on the responsibility of maintaining the new repository
```

Comment:
> with the assumption that any committer who approves is taking on the responsibility of maintaining the new repository

What does this mean?

Let's take https://github.com/cilium/state-writer as an example. After an initial flurry of activity, development goes quiet.

I suspect this requirement may lead to one of a couple of results:
- One, the committers who approve practically take no steps to maintain the repository. The person requesting the repository will maintain it, or it will become unmaintained after initial development.
- Two, committers may avoid approving PRs because the process says they're now responsible for this thing that they have no involvement in. Unless you can convince two committers to take responsibility, the request may end up in deadlock.

Reply:
Yes I can see how that could lead to problems, I can take it out. Perhaps just having two people approve of the new repository is sufficient

Reply:
Maybe "maintain" is too strong a word here and something more like be the point of contact for would be better. Basically two people saying yes this is our idea contact us if you want to get involved

Reply:
If we look at the list of all the repositories we have today, does that model make sense and how do you think we would document these owners?

For core repositories or other repositories where we're expecting active engagement from multiple community members, I think this totally makes sense. At the same time, I think there's a long tail of repositories where this doesn't really make sense and it's broadly fine for community members to just reach out on the #development Slack channel to ask about status / how to get involved. Typically someone will be around to redirect the queries or pull in someone who knows about the repo.

I'm not averse to having some documentation like this, but I think it would help these owners to define clearly what the expectations would be when they sign up to own such a repository. If it's just "contact us if you want to contribute to the new repository, and we'll figure out how to make that happen" then I think that's very reasonable, but let's state that clearly.